src/app/container/view/view.component.ts

```diff
@@ -200,8 +197,20 @@ export class ViewContainerComponent extends View implements OnInit, AfterViewChe
         this.mode = mode;
       }
     );
+
     this.unsavedVersion = Object.assign({}, this.version);
-    this.tool = this.communicatorService.getTool();
+    this.containerService.tool$.subscribe(tool => {
+    this.tool = tool;
```

Comment:
Comment on this, is this making a tool editable if it is published or just if this tag editor is in the public section of the GUI?